src/tools/compiletest/src/runtest/incremental.rs

```diff
@@ -0,0 +1,128 @@
+use super::{TestCx, WillExecute};
+use crate::errors;
+
+// FIXME(jieyouxu): `run_rpass_test` got hoisted out of this because apparently valgrind falls back
+// to `run_rpass_test` if valgrind isn't available, which is questionable, but keeping it for
+// refactoring changes to preserve current behavior.
```

Comment:
Remark: FIXME because valgrind just runs rpass if valgrind isn't available, which is highly questionable.